.kokoro/common.cfg

```diff
@@ -11,7 +11,7 @@ action {
 gfile_resources: "/bigstore/cloud-devrel-kokoro-resources/trampoline"
 
 # Use the trampoline script to run in docker.
-build_file: "nodejs-document-ai/.kokoro/trampoline.sh"
+build_file: "nodejs-documentai/.kokoro/trampoline.sh"
```

Comment:
This seems to revert our work from the other day that updated `package.json` to have `nodejs-document-ai` as the library name. Is there a chance you were working from an older branch?